applications/luci-app-smartdns/root/usr/share/rpcd/acl.d/luci-app-smartdns.json

```diff
@@ -4,9 +4,8 @@
 		"read": {
 			"file": {
 				"/etc/smartdns/*": [ "read" ],
-				"/usr/sbin/iptables -t nat -nL PREROUTING": [ "exec" ],
-				"/usr/sbin/ip6tables -t nat -nL PREROUTING": [ "exec" ],
-				"/usr/sbin/smartdns": [ "exec" ]
+				"/usr/sbin/smartdns": [ "exec" ],
+				"/etc/init.d/smartdns restart" : [ "exec" ]
```

Comment:
You should move this permission into the `"write": { "file": { ... } }` scope as it affects the system state.